src/bicep-types/src/writers/markdown.ts

```diff
@@ -176,6 +196,14 @@ export function writeMarkdown(types: BicepType[], fileHeading?: string) {
         md.writeBullet("Valid Scope(s)", `${getScopeTypeLabels(resourceType.scopeType, [resourceType.readOnlyScopes, 'ReadOnly']).join(', ') || 'Unknown'}`);
         writeComplexType(types, types[resourceType.body.index], nesting, false);
 
+        if (resourceType.functions) {
+          for (const functionName in resourceType.functions) {
+            const { type } = resourceType.functions[functionName];
+            writeFunctionType(functionName, types[type.index] as FunctionType, nesting + 1);
+            return;
```

Comment:
@anthony-c-martin Should the return statement here be outside the for loop? I think this would only write the first function type on a resource otherwise. Unless there's something I'm missing

Reply:
Oops, good spot. Fix: #421